packages/@coorpacademy-components/package.json

```
@@ -13,7 +13,9 @@
   ],
   "scripts": {
     "build:commonjs": "cross-env BABEL_ENV=commonjs babel src --out-dir lib -s --copy-files --no-copy-ignored --ignore \"**/test\" --extensions \".ts,.tsx,.js\"",
-    "build:es": "cross-env BABEL_ENV=es babel src --out-dir es -s --copy-files --no-copy-ignored --ignore \"**/test\" --extensions \".ts,.tsx,.js\"",
+    "build:es:babel": "cross-env BABEL_ENV=es babel src --out-dir es -s --copy-files --no-copy-ignored --ignore \"**/test\" --extensions \".ts,.tsx,.js\"",
+    "build:es:types": "tsc -p tsconfig.es.json",
```

```suggestion
    "build:es:types": "tsc --module es2020 --outDir es --emitDeclarationOnly --noEmit false",
```

Il faudrait generer aussi les types pour commonjs

---

ok 👍